### PR TITLE
Make bookings

### PR DESCRIPTION
**Pull request by Brar-Paul**
Adding functionality to create bookings, made form for bookings w/ datepicker and required JS, made changes to DB schema

### CODE REVIEW

File: app/controllers/workshops_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def new
   def create
     @workshop = Workshop.new(workshop_params)
     @workshop.user = current_user
+    raise
```
Comment (deluzie) [inline app/controllers/workshops_controller.rb]: there is still a raise here, i think you forgot to take it out :)

File: app/models/booking.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :workshop
-  belongs_to :review
+  has_many :reviews
```
Comment (deluzie) [inline app/models/booking.rb]: unsure about that, one booking only has one review, right? so workshops can have multiple reviews but only one for each booking i would think.

Comment (Brar-Paul) [reply app/models/booking.rb]: Yes you're right, I was mixing up bookings and workshops. It should be a one to one relationship. So I think review will belongs_to :booking and then the booking model will have no mention of review, right?

File: app/controllers/workshops_controller.rb (hunk 1 above)
Comment (Brar-Paul) [reply app/controllers/workshops_controller.rb]: My bad, I removed it.

**Review by deluzie — APPROVED** [review]
looks all good now!

**Pull request merged**